examples/Directory.Packages.props

```diff
@@ -1,6 +1,6 @@
 <Project>
   <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Packages.props, $(MSBuildThisFileDirectory)..))" />
   <ItemGroup>
-    <PackageVersion Update="System.Text.Json" Version="6.0.5" />
+    <PackageVersion Update="System.Text.Json" Version="7.0.1" />
```

Comment:
Why do we need to update the version for `examples/`?

Reply:
I am thinking to make this aligned with the version in Directory.Packages.props.

Reply:
This is `Directory.Packages.props`. What does it have to align with?

Reply:
reverted